src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -3031,10 +3032,32 @@ TEST_F(PolicyHandlerTest, GetAppPropertiesStatus_NicknameChanged_SUCCESS) {
             policy_handler_.GetAppPropertiesStatus(properties, kPolicyAppId_));
 }
 
-TEST_F(PolicyHandlerTest, GetAppPropertiesStatus_NicknameNotChanged_SUCCESS) {
+TEST_F(PolicyHandlerTest,
+       GetAppPropertiesStatus_RemoveNickname_NicknamesChanged) {
   ChangePolicyManagerToMock();
 
-  const std::string kFakeNickname = "fake_nickname";
+  smart_objects::SmartObject properties;
+  properties[strings::app_id] = kPolicyAppId_;
+  properties[strings::nicknames] =
+      smart_objects::SmartObject(smart_objects::SmartType_Array);
+  properties[strings::nicknames].asArray()->push_back(
+      smart_objects::SmartObject(kFakeNickname));
+
+  auto nicknames = std::make_shared<policy::StringArray>(2, kFakeNickname);
```

Comment:
@SNiukalov Missed required header file inclusion

Reply:
@AByzhynar 
Thanks, fixed in:https://github.com/smartdevicelink/sdl_core/pull/3336/commits/19023b6acd3686175ff83e5c6e3ab453c09d6b35